authors/apps/article/views.py

```diff
@@ -1,18 +1,17 @@
-
-# Create your views here.
-from rest_framework.exceptions import NotFound
 from django_filters.rest_framework import DjangoFilterBackend
-from rest_framework import generics, status
-from rest_framework.generics import get_object_or_404
-from rest_framework.permissions import AllowAny, IsAuthenticated
+from rest_framework import filters, generics, status
+from rest_framework.exceptions import NotFound
+from rest_framework.generics import Http404, get_object_or_404
```

Comment:
I suggest that you remove one line of `generics` importations because you first import `generics` as a whole and import other classes again from `generics`.

Reply:
Hello Lule, we would have to figure whose work added line 4 so that they can change it and call Http404 as generics.Http404 and get_object_or_404 as generics.get_object_or_404